hook-docker/main.go

```diff
@@ -38,28 +34,23 @@ func main() {
 	cmdLines := strings.Split(string(content), " ")
 	cfg := parseCmdLine(cmdLines)
 
-	path := fmt.Sprintf("/etc/docker/certs.d/%s/", cfg.registry)
-
-	// Create the directory
-	err = os.MkdirAll(path, os.ModeDir)
-	if err != nil {
-		panic(err)
-	}
-	// Download the configuration
-	err = downloadFile(path+"ca.crt", cfg.baseURL+"/ca.pem")
-	if err != nil {
-		panic(err)
-	}
-	fmt.Println("Downloaded the repository certificates, starting the Docker Engine")
+	fmt.Println("Starting the Docker Engine")
 
 	d := dockerConfig{
 		Debug:     true,
 		LogDriver: "syslog",
 		LogOpts: map[string]string{
 			"syslog-address": fmt.Sprintf("udp://%v:514", cfg.syslogHost),
 		},
+		InsecureRegistries: cfg.insecureRegistries,
+	}
+	path := "/etc/docker"
+	// Create the directory for the docker config
+	err = os.MkdirAll(path, os.ModeDir)
+	if err != nil {
+		panic(err)
 	}
-	if err := d.writeToDisk("/etc/docker/daemon.json"); err != nil {
+	if err := d.writeToDisk(filepath.Join(path, "daemon.json")); err != nil {
 		fmt.Println("Failed to write docker config:", err)
```

Comment:
Shouldn't we have a `return` after this?

Reply:
ah good catch, thanks! will update